lisa/tools/copy_back.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import os
+from pathlib import PurePath
+
+from lisa.executable import Tool
+from lisa.tools.ls import Ls
+
+
+class CopyBack(Tool):
+    @property
+    def command(self) -> str:
+        return ""
+
+    @property
+    def can_install(self) -> bool:
+        return False
+
+    def copy_dir(
+        self,
+        src: PurePath,
+        dest: PurePath,
+        sudo: bool = False,
```

Comment:
this sudo is not used, so this method doesn't support sudo.

Reply:
I forgot use it in `ls`. Will add it there